docs/tags.md

```diff
@@ -32,6 +32,9 @@ In our Cypress and Cucumber setup, tags are used to categorize and selectively r
   <br>
 
 - **@group-B:** This tag is used for grouping the certain feature files into 'group-B' for the parallel tests execution in the CI/Github actions.
+
+- **@group-C: ....so on**
```

Comment:
instead of mentioning every group. make it generalised and mention as an example how it would be 

```md

- **@group-*: ** Used to group test files for parallel execution in CI. This helps divide tests across multiple runners for faster test execution. 
Examples: @group-A, @group-B, @group-C, etc.
```